libcxx/utils/ci/buildkite-pipeline.yml

```diff
@@ -28,916 +28,520 @@ env:
     # LLVM RELEASE bump remove compiler ToT - 3, e.g. "Clang 15"
     LLVM_HEAD_VERSION: "18"   # Used compiler, update POST-BRANCH.
     GCC_STABLE_VERSION: "13"
-steps:
-  #
```

Comment:
Did you intentionally drop that comment?

Reply:
Sorry, which comment? All I see is an empty comment line in the diff.

I did the transformation with hand-rolled code, and the YAML parser didn't maintain comments in its representation.

Reply:
This one:

```
#
# General testing with the default configuration, under all the supported
# Standard modes, with Clang and GCC. This catches most issues upfront.
# The goal of this step is to catch most issues while being very fast.
#
```

It was at the top before the first section of tests.

Reply:
Re added.